listenbrainz/tests/integration/test_user_timeline_event_api.py

```diff
@@ -815,3 +815,123 @@ def test_get_cb_review_events(self, mock_requests):
         self.assertEqual(metadata_3["rating"], review_event_3["metadata"]["rating"])
         self.assertEqual(metadata_3["text"], review_event_3["metadata"]["text"])
         self.assertEqual(review_mbid_3, review_event_3["metadata"]["review_mbid"])
+
```

Comment:
These tests look good we should probably also add 1 more where the user tries to recommend to a user they don't follow.